generators/app/templates/_generator.yml

```
@@ -0,0 +1,21 @@
+name: Generator
+on:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
```

Why ignoring dependabot?
```suggestion
  push:
```

---

Good question.